.github/workflows/release.yaml

```diff
@@ -0,0 +1,62 @@
+name: Release a new version/tag
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: Version/tag to release
+        required: true
+        type: string
+
+  # push:
```

Comment:
@aradjdi What about using the git tag as a version rather than the input? I think it's better because we can fixate the release tag with the code version tag. (unless you're using the input for testing only)

Reply:
It's also for testing, I'll try later to launch this job automatically when tag event has been fire